include/aie/Dialect/AIE/IR/AIETargetModel.h

```diff
@@ -463,29 +462,28 @@ class IPUTargetModel : public AIE2TargetModel {
   }
 };
 
-} // namespace AIE
-} // namespace xilinx
+} // namespace xilinx::AIE
 
 namespace llvm {
-using namespace xilinx::AIE;
```

Comment:
This one seems OK?

Reply:
Yea I don't know but it doesn't smell right - adding a `using` for someone else's namespace in one of our namespaces seems fine - the reverse not so much.

Reply:
Yeah, playing further I think it's just not right, even within the namespace.